js/app/delivery/AddressBook.js

```diff
@@ -1,20 +1,19 @@
 import React, { useState, useRef, useEffect } from 'react'
 import { render } from 'react-dom'
 import _ from 'lodash'
-import { Popover, Form, Input, Button } from 'antd'
+import {  Input, Button } from 'antd'
 import { UserOutlined, PhoneOutlined, StarOutlined } from '@ant-design/icons'
 import { useTranslation } from 'react-i18next'
-import classNames from 'classnames'
 import { parsePhoneNumberFromString } from 'libphonenumber-js'
 import Modal from 'react-modal'
 
 import AddressAutosuggest from '../components/AddressAutosuggest'
 import { getCountry } from '../i18n'
 
 import './AddressBook.scss'
-import { SavedAddressesBox } from './SavedAddressBox'
+import SavedAddressesBox from './SavedAddressBox'
```

Comment:
If you ask me, I'd prefer named imports 😄

Reply:
changed